tools/src/main/kotlin/org/dashevo/tools/DashPayWallet.kt

```diff
@@ -0,0 +1,291 @@
+/**
+ * Copyright (c) 2020-present, Dash Core Group
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package org.dashevo.tools
+
+import com.google.common.base.Stopwatch
+import io.grpc.StatusRuntimeException
+import org.bitcoinj.core.Context
+import org.bitcoinj.core.PeerGroup
+import org.bitcoinj.crypto.KeyCrypterException
+import org.bitcoinj.evolution.EvolutionContact
+import org.bouncycastle.crypto.params.KeyParameter
+import org.dashevo.dashpay.*
+import org.dashevo.dashpay.Profile
+import org.dashevo.dpp.document.Document
+import org.dashevo.dpp.toBase58
+import org.dashevo.platform.DomainDocument
+import org.slf4j.LoggerFactory
+import java.lang.Long.max
+import java.util.concurrent.atomic.AtomicBoolean
+
+class DashPayWallet (val blockchainIdentity: BlockchainIdentity, val peerGroup: PeerGroup?, val password: String? = null) {
```

Comment:
This class is similar to what is in the PlatformRepo.kt file in `dash-wallet` at least as far as what we need for `wallet-tool`.

It may make sense to move these methods into other classes (even to BlockchainIdentity).  Having these method's available in android-dashpay makes this library more robust and supportive of the DashPay contract.